### DISCUSSION TITLE
Support for TimescaleDB compressed tables in the postgres scanner

### DISCUSSION THREAD

**Discussion by andnofence** [duckdb/duckdb#8622] [Ideas]
I wanted to try out DuckDB against our TimescaleDB database. TimescaleDB is great for the IoT data we collect, but it's not ideal as an analytics database. So when I found the DuckDB Postgres scanner I was excited as it seems we could have our cake and eat it to. Unfortunately it seems the two tools are not compatible, when trying to read data from a TimescaleDB table that uses compression I get the following error:

> Error: Invalid Error: IO Error: Unable to query Postgres: ERROR:  transparent decompression only supports tableoid system column

So I'm wondering if you have looked into making the scanner compatible with TimescaleDB compressed tables?

**Comment by Tishj** [comment]
Sounds like this belongs here:
<https://github.com/duckdblabs/postgres_scanner>

**Reply by andnofence** [reply]
Thanks, didn't realize they were separate repos, I've moved it there: https://github.com/duckdblabs/postgres_scanner/issues/104
